### ISSUE TITLE
Can not Close Branches in 4.1.86

### ISSUE THREAD

**Issue by jonl-percsolutions-com** [FlutterFlow/flutterflow-issues#3745]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When attempting to delete a branch, I am seeing an exception in the console and the delete.

### Expected Behavior

Receive the popup for confirming closing a branch.
Successfully close the branch.

### Steps to Reproduce

1. Create a new branch or go to an existing branch
2. Open the branch dialog
3. Select the close option

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT40tfqAuLlIotdI+s+JK/JUoAE4KWI7XLgZq9xRaCosfZT2BLMcec/RVERCQu2iTGw3MGaahWoW3NqNvobqUMACByqZbJs4zatqUBXybHKnaba7FJWgeXJ8Ep9MBXKF1sKz3RMlJtViSHtg60+qF9nIEj7sQeL1Clc4Dq/LZO4=

### Visual documentation

Receive the following error on clicking the button:

![image](https://github.com/user-attachments/assets/bbe0fc2a-3e4d-4214-b93e-98f9b4e1c57c)

### Environment

```markdown
- FlutterFlow version: 4.1.86+
- Platform:Windows/Web
- Browser name and version: Chrome/Edge
- Operating system and version affected:
```

### Additional Information

This worked prior on 4.1.85 or less from our recollection.

**Comment by rzambroni** [comment]
Hi @jonl-percsolutions-com,

Thank you so much for the detailed report. I was able to replicate the issue on my end and will be forwarding it to the team for further review. We’ll keep you updated as soon as we have more information.

Thanks again for bringing this to our attention!

**Comment by jonl-percsolutions-com** [comment]
@victoriahuang1 , I see this was closed yesterday as completed and build 41.87+ came out yesterday, does that contain this fix or will it be a newer build?  We still can not close branches in 4.1.87+.

It would be great if changelogs were released for each point release.

**Comment by victoriahuang1** [comment]
@jonl-percsolutions-com it should go out with the next deploy.

Thanks for the feedback. Yes, I agree that it would be great to increase visibility on what is released
